README.md

```diff
@@ -117,6 +121,33 @@ flutter build ipa
 
 - The Xcode build archive (.xcarchive file) generated can be found inside `[project]/build/ios/archive/` directory and an App Store app bundle (.ipa file) in `[project]/build/ios/ipa/`
 
+## How to Use
+
+### CLN gRPC
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.grpc`
+- Provide the directory path consisting of the TLS Certificate\
+  Example: `/ssd/home/usr/lnd`. The lnd folder consists of the TLS Cert file
+- Enter the host alongside the port of your lightning node. You can find it by running `lightning-cli getinfo`\
+  Example: `120.154.200.80:19735`
+
+
+### LN Lambda
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.lnlambda`
+- Type in the node id and host (alongside the port). You can find both the values by running `lightning-cli getinfo`
+- For the Lambda Server field, type in `https://lnlambda.lnmetrics.info`
```

Comment:
Maybe you should make also another Pr to make the lnlambda.lnmetrics.info a default option?

Reply:
Yes can do. Should I do that after this PR is merged?

Reply:
yes another PR sounds good